libminifi/include/core/state/nodes/MetricsBase.h

```diff
@@ -108,27 +104,33 @@ class ObjectNode : public ResponseNode {
     nodes_.push_back(node);
   }
 
-  virtual std::string getName() const {
+  const std::vector<std::shared_ptr<ResponseNode>>& get_child_nodes() const {
```

Comment:
I see that the casing is mixed in this class, IMO we should go with camelCase for new methods

Reply:
Removed `get_child_nodes` in 40b469f11033f229a9237afd0b3ba397cb2e316d because it is not used anymore.